### DISCUSSION TITLE
Toggle::dehydratedWhenHidden does not exist.

### DISCUSSION THREAD

**Discussion by khalilKhassep** [awcodes/filament-tiptap-editor#388] [Q&A]
I found this exception fired when i click inset image button and i dont know its a bug or what else i solve it by changing dehydratedWhenHidding to dehydrated(config('filament-tiptap-editor.disable_link_as_button')) and i think this not a dynmic soultution becuse its a static alwys have the save value. 
How we can solve this ? 
![image](https://github.com/awcodes/filament-tiptap-editor/assets/47596595/f09c785b-aa28-4687-a67f-e0582f541718)

**Comment by awcodes** [comment]
You need to update Filament.
